### PR TITLE
Explicate range of key weights

### PR DESCRIPTION
**Pull request by ire-and-curses**
Experimentally, key weights seem to be allowed for 0-255. Make this explicit in the doc.

### CODE REVIEW

File: guides/concepts/list-of-operations.md [hunk 1]
```diff
@@ -221,12 +221,12 @@ Parameters:
...
+|Master weight| integer| A number from 0-255 representing the weight of the master key. This account may also add other keys with which to sign transactions using `signer` below.|
 |Low threshold| integer| A number from 0-255 representing the threshold this account sets on all operations it performs that have [a low threshold](./multi-sig.md).|
 |Medium threshold| integer| A number from 0-255 representing the threshold this account sets on all operations it performs that have [a medium threshold](./multi-sig.md).|
 |High threshold| integer| A number from 0-255 representing the threshold this account sets on all operations it performs that have [a high threshold](./multi-sig.md). |
 |Home domain| string| Sets the home domain of an account. See [Federation](./federation.md).|
-|Signer| {Public Key, weight}| Add, update, or remove a signer from an account.  The signer is deleted if the weight is 0.|
+|Signer| {Public Key, weight}| Add, update, or remove a signer from an account. Signer weight is a number from 0-255. The signer is deleted if the weight is 0.|
```
Comment (morleyzhi) [inline guides/concepts/list-of-operations.md]: Are 0 and 255 valid numbers or no?

Comment (ire-and-curses) [reply guides/concepts/list-of-operations.md]: Yeah, both are valid.

Comment (morleyzhi) [reply guides/concepts/list-of-operations.md]: (Sorry, I meant to convey that the documentation should probably be specific about whether it's inclusive or exclusive.)

Comment (ire-and-curses) [reply guides/concepts/list-of-operations.md]: ok, updated

**Comment by tomerweller** [comment]
The master weight description might be a bit confusing (both before and after).

> A number from 0-255 (inclusive) representing the weight of the master key.

This is simple and correct. I'd keep that. 

>This account may also add other keys with which to sign transactions using `signer` below.

Any transaction that meets the "high" weight threshold requirements can do that, this is not special for the master signer. 

> If the weight of the master key is ever updated to 0, the master key is considered to be an invalid key and cannot be used to sign transactions (even for operations with a threshold value of 0)

"Invalid" might be a bit confusing as the weight can be restored using a transaction with other signers passing the "high" threshold.

**Comment by ire-and-curses** [comment]
@tomerweller Thanks, that's interesting. I actually pulled that text from https://www.stellar.org/developers/guides/concepts/multi-sig.html. Indeed I understood it to mean that the key could never have its weight modified again. I guess we should improve this wording in both places.

**Pull request merged**